docs/management_api.yml

```diff
@@ -81,6 +295,10 @@ definitions:
       error:
         description: Description of the error.
         type: string
+      request_id:
```

Comment:
The RestErrWithLogMsg from go-lib-micro/rest_utils is adding request_id to response body.